src/effects/effectslot.cpp

```diff
@@ -146,9 +147,7 @@ void EffectSlot::loadEffect(EffectPointer pEffect) {
         m_pControlNumParameters->forceSet(pEffect->numKnobParameters());
         m_pControlNumButtonParameters->forceSet(pEffect->numButtonParameters());
 
-        // Enabled is a persistent property of the effect slot, not of the
-        // effect. Propagate the current setting to the effect.
-        pEffect->setEnabled(m_pControlEnabled->get() > 0.0);
+        pEffect->updateEngineState();
```

Comment:
No, the original line was a no-op because setEnabled() checks if the parameter passed to it is equal to its private enabled boolean. pEffect->updateEngineState() is necessary to sync the state of the EffectSlot and the Effect when a new effect is loaded to an enabled effect slot now that Effects are disabled by default.